src/ResourceManager/Profile/Commands.Profile/Commands.Profile.csproj

```diff
@@ -9,9 +9,8 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Microsoft.Azure.Commands.Profile</RootNamespace>
     <AssemblyName>Microsoft.Azure.Commands.Profile</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
+    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
```

Comment:
Please remove this change, and the change in the file below

Reply:
Done